components/form-builder/app/navigation/LeftNavLink.tsx

```diff
@@ -27,6 +27,7 @@ export const LeftNavLink = ({ children, href }: { children: ReactElement; href:
         className={`${
           active ? "font-bold" : ""
         } group no-underline rounded block w-38 py-1 mb-2 pl-2 pr-0 laptop:pr-2 text-black-default hover:text-blue-hover visited:text-black-default focus:text-white-default focus:bg-blue-hover active:no-underline active:bg-blue-hover active:text-white-default !shadow-none`}
+        {...(active && { "aria-current": "page" })}
```

Comment:
AT support for On/Off with ARIA attributes generally works more consistently by removing the attribute for the Off/false state. For aria-current, our use is only "page" or false, so it's either on the element or just removed altogether